checkstyle/import-control.xml

```diff
@@ -93,13 +93,19 @@
 
     <subpackage name="record">
       <allow pkg="net.jpountz" />
+      <allow pkg="org.apache.kafka.common.header" />
```

Comment:
I was purposely wanting to avoid that and use packaging to keep things tidy, else the common package level (one level up) would become a dumping ground over a period, if that approach was constantly taken.

This is simply allowing package
org.apache.kafka.common.record 

import
org.apache.kafka.common.header